src/main/java/com/avail/optimizer/jvm/JVMTranslator.java

```diff
@@ -1799,14 +1802,7 @@ void generateRunChunk ()
 							load(
 								method,
 								((L2ReadIntOperand) operand).register());
-							method.visitMethodInsn(
-								INVOKESTATIC,
-								getInternalName(Integer.class),
-								"valueOf",
-								getMethodDescriptor(
-									getType(Integer.class),
-									INT_TYPE),
-								false);
+							javaUnboxIntegerMethod.generateCall(method);
```

Comment:
Wow, much cleaner semantics. It actually says what the desired operation and outcome is. `valueOf` is really feebly named.